pkg/plugins/plugins.go

```diff
@@ -89,21 +98,34 @@ func (p PluginDTO) IsSecretsManager() bool {
 	return p.JSONData.Type == SecretsManager
 }
 
-func (p PluginDTO) IncludedInSignature(file string) bool {
-	// permit Core plugin files
-	if p.IsCorePlugin() {
-		return true
+type File interface {
+	fs.File
+	io.Seeker
+}
+
+func (p PluginDTO) File(name string) (File, error) {
```

Comment:
It's a tad bit weird that the `PluginDTO` exposes these methods. DTO's are normally just data/state carriers. Something to consider for the future perhaps?

Reply:
Yeah in retrospect DTO was a bad name. This is really just the Grafana model for a plugin so we might just rename the internal one instead and call this Plugin. Will add this along with the other comments to the doc we have 😁